### PR TITLE
Transformation of Inherited Indo-European Phraseology

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/transformation-of-inherited-indo-european-phraseology_olav-hackstein.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Skt. dāyāda- 'Eating Away at the Inherited/Entrusted': The Transformation of Inherited Indo-European Phraseology in the Buddhist Legend of Ajātaśatru"
```
Comment (buddhist-uni) [inline _content/articles/transformation-of-inherited-indo-european-phraseology_olav-hackstein.md:2]: Put something in *s to make it italic: `*dāyāda-*`

File: _content/articles/transformation-of-inherited-indo-european-phraseology_olav-hackstein.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+authors:
+  - "Olav Hackstein"
+external_url: "https://poj.peeters-leuven.be/content.php?url=article&id=3134549&journal_code=JIABS&download=yes"
+drive_links:
+  - "https://drive.google.com/file/d/1NVzUGptMFQaElYOfbL64yDfZb5p90DSj/view?usp=sharing"
+course: karma
+tags:
+  - language
```
Comment (buddhist-uni) [inline _content/articles/transformation-of-inherited-indo-european-phraseology_olav-hackstein.md:10]: let's maybe also add the `pali-dictionaries` tag? Not exactly right, but I don't yet have a Pali Philology tag, so the lexicographic tag will have to do double duty for now...

And possibly also the `historiography` and `mythology` tags, as it's such a fun example (to me at least!)

File: _content/articles/transformation-of-inherited-indo-european-phraseology_olav-hackstein.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+journal: jiabs
+volume: 38
+pages: "391--400"
+---
+
+>The same Indo-European metaphor for
+abusing paternal property is traceable in the Middle Iranian and Indic
+tradition, ranging from Vedic to (Buddhist) Sanskrit dāyāda-.
+
+This article focuses on the philology of a Sanskrit term and its relation to the Buddhist story of Ajātaśatru and how it is understood. The study focuses on the translation of the term dāyāda and how Indo-European metaphors, mainly Homeric, informed the translation and understanding of the term into other languages, particularly West Tocharian.
```
Comment (buddhist-uni) [inline _content/articles/transformation-of-inherited-indo-european-phraseology_olav-hackstein.md:21]: The reason I put this in karma is that karma is described as *dāyādo*:

> Kammassakomhi kamma-**dāyādo** kamma-yoni kamma-bandhu kamma-paṭisaraṇo.

Would be good to remind the reader of the importance of this term in the description! Give them a reason to read this obscure history 😉

Comment (JJM317) [reply _content/articles/transformation-of-inherited-indo-european-phraseology_olav-hackstein.md]: Sounds good. I changed it up a bit. See what you think.

**Review by buddhist-uni — APPROVED** [review]
Excellent! Much better, thanks 😊

**Pull request merged**